### PR TITLE
multiple projects addition

### PR DESCRIPTION
**Pull request by xinity**
This a a multiple projects Addition:
- DockerSpec
- TravisCI
- ConcourseCI
- hadolint
- SaltStack Docker module

Hoping this helps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [Skipper](https://github.com/Stratoscale/skipper) - Easily dockerize your Git repository by [@Stratoscale](https://github.com/Stratoscale)
 * [SwarmCI](https://github.com/ghostsquad/swarmci) - Create a distributed, isolated task pipeline in your Docker Swarm.
 * [Watchtower](https://github.com/v2tec/watchtower) - Automatically update running Docker containers by [@CenturyLinkLabs][CenturyLinkLabs]
+* [DockerSpec](https://github.com/zuazo/dockerspec) - A small Ruby Gem to run RSpec and Serverspec, Infrataster and Capybara tests against Dockerfiles or Docker images easily.
```
Comment (veggiemonk) [inline README.md]: This should be in lexical order (English letter order)

File: README.md [hunk 2]
```diff
@@ -144,6 +145,10 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [Semaphore CI](https://semaphoreci.com/product/docker) — A high-performance cloud solution that makes it easy to build, test and ship your containers to production.
 * [Shippable](https://app.shippable.com/) - A SaaS platform for developers and DevOps teams that significantly reduces the time taken for code to be built, tested and deployed to production.
 * [IBM DevOps Services](https://hub.jazz.net) - Continuous delivery using a pipeline deployment onto IBM Containers on Bluemix.
+* [TravisCI](https://travis-ci.org/) - A Free github projects continuous integration Saas platform for developers and Devops.
+* [ConcourseCI](https://concourse.ci) - A CI SaaS platform for developers and DevOps teams pipeline oriented.
```
Comment (veggiemonk) [inline README.md]: This should be in lexical order (English letter order)

File: README.md [hunk 3]
```diff
@@ -256,6 +261,7 @@ Services to securely store your Docker images.
 * [dockerfile_lint](https://github.com/projectatomic/dockerfile_lint) - A rule-based 'linter' for Dockerfiles by [@redhataccess](https://github.com/redhataccess)
 * [Lorry](https://lorry.io/) - Lorry is a docker-compose.yml validator and composer by [@CenturyLinkLabs][CenturyLinkLabs]
 * [Whale-linter](https://github.com/jeromepin/whale-linter) - A simple and small Dockerfile linter written in Python3+ without dependencies.
+* [Hadolint](https://github.com/lukasmartinelli/hadolint) - A Dockerfile linter written in Haskell
```
Comment (veggiemonk) [inline README.md]: This should be in lexical order (English letter order)

File: README.md [hunk 4]
```diff
@@ -276,6 +282,7 @@ Services to securely store your Docker images.
 * [Turbo](https://ramitsurana.github.io/turbo/) - Simple and Powerful utility for docker. By [@ramitsurana][ramitsurana]
 * [Vagrant - Docker provider](https://www.vagrantup.com/docs/docker/basics.html) - Good starting point is [vagrant-docker-example](https://github.com/bubenkoff/vagrant-docker-example) by [@bubenkoff](https://github.com/bubenkoff)
 * [Vessel](https://github.com/awvessel/vessel) - Automates the setup & use of dockerized development environments by [@awvessel](https://github.com/awvessel)
+* [SaltStack Docker module](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.dockerng.html#module-salt.modules.dockerng) - SaltStack Docker module
```
Comment (veggiemonk) [inline README.md]: This should be in lexical order (English letter order)

**Comment by xinity** [comment]
all done :)

File: README.md [hunk 5]
```diff
@@ -254,6 +259,7 @@ Services to securely store your Docker images.
 
 ## Linter / Validator
 * [dockerfile_lint](https://github.com/projectatomic/dockerfile_lint) - A rule-based 'linter' for Dockerfiles by [@redhataccess](https://github.com/redhataccess)
+* [Hadolint](https://github.com/lukasmartinelli/hadolint) - A Dockerfile linter written in Haskell
```
Comment (devhkr) [inline README.md]: Could you please add the author's GitHub username with respective link after the description? see an example above.

File: README.md [hunk 6]
```diff
@@ -127,6 +127,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [Cyclone](https://github.com/caicloud/cyclone) - A cloud native CI/CD platform built for container workflow by [@caicloud](https://github.com/caicloud).
 * [Docker plugin for Jenkins](https://github.com/jenkinsci/docker-plugin/) - The aim of the docker plugin is to be able to use a docker host to dynamically provision a slave, run a single build, then tear-down that slave.
 * [Dockunit](https://github.com/dockunit/platform) - Docker based integration tests. A simple Node based utility for running Docker based unit tests. By [@dockunit](https://github.com/dockunit)
+* [DockerSpec](https://github.com/zuazo/dockerspec) - A small Ruby Gem to run RSpec and Serverspec, Infrataster and Capybara tests against Dockerfiles or Docker images easily.
```
Comment (devhkr) [inline README.md]: Could you please add the author's GitHub username with respective link after the description? see an example above.

**Comment by xinity** [comment]
oups, haven't seen the comment :( sorry guys!

thanks a lot @devhkr fixing my PR, should i update mine so ?

**Comment by devhkr** [comment]
Welcome back @xinity 😄

No further commit is needed from you, as on PR #333 your changes are included there.
In addition, with have an issue with a build error where we have to look closer.

Again, thanks for your contribution and if you find good projects don't hesitate to contribute here which we really appreciate.

Have a good day left.

The awesome docker team
@vegasbrianc @gesellix @dmitrytokarev @veggiemonk @agebhar1 @devhkr

**Comment by veggiemonk** [comment]
@xinity thanks again for your contribution!

**Pull request merged**